### ISSUE TITLE
Teleop controls overhaul

### ISSUE THREAD

**Issue by MichGedd** [FRC1360/ChargedUp2023#46]
**Is your feature request related to a problem? Please describe.**
Communication between Driver and Operator is slow. We should try to consolidate as many controls as possible onto the Driver's controller.

**Describe the solution you'd like**
The following control scheme has been suggested by Drive Team:
- Move intake to driver controller
- Move outtake to driver controller (done)
- Move "Go To Home" to driver controller
- Move "Go To Active Setpoint" to driver controller

The operator controller will select a setpoint (such as Score High Cone, Score Mid Cone, etc) as the "active setpoint". When the driver presses the "Go To Setpoint" button, the Arm will go to whatever setpoint is the "active setpoint".

TLDR - Operator controls where to move the arm to, Driver controls when to move the arm.

**Describe alternatives you've considered**
Another alternative considered is move everything to a single controller, but that may be too much for the driver to do.

**Additional context**
N/A
